runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpTrackerPlugin.java

```diff
@@ -39,9 +39,9 @@
 import net.runelite.client.task.Schedule;
 
 @PluginDescriptor(
-	name = "XP Tracker plugin"
+	name = "Xp tracker plugin"
```

Comment:
Keep this as "XP"

Reply:
User-facing (or potentially user-facing 'xp' has been changed to 'XP'): 96b4f6e
Class names and such will remain 'Xp' to follow typical Java conventions.